exporters/scss/src/js/generators/typography.ts

```diff
@@ -41,7 +41,7 @@ export function generateTypography(
     } else {
       fontWeight = subfamily;
     }
-    const lineHeight = Math.round((token.value.lineHeight.measure / 100) * 1000) / 1000;
+    const lineHeight = token.value.lineHeight ? Math.round((token.value.lineHeight.measure / 100) * 1000) / 1000 : '0';
```

Comment:
Are you sure we want to set default `line-height` to `0`? I would prefer recommended `1.5` or something like that.

Reply:
We don't actually print 0. When the line height is zero we don't print it. But I understand it is confusing, so I improved it. Anyway, the best scenario is that every typography style has a line height set.